config/job.js

```diff
@@ -102,6 +108,8 @@ module.exports = {
     secret: SCHEMA_SECRET,
     secrets: SCHEMA_SECRETS,
     settings: SCHEMA_SETTINGS,
+    sourcePath: SCHEMA_SOURCEPATH,
+    sourcePaths: SCHEMA_SOURCEPATHS,
```

Comment:
Why do we have `sourcePath` and `sourcePaths`? Is this for backward compatibility?

Reply:
It's a single `sourcePath` vs multiple `sourcePaths`. But in the yaml config, it only allows `sourcePaths`

Btw, can we change it to `srcpaths`? Then we don't need to get into the camel vs snake case

Reply:
why do we have `sourcePath`, it should only be `sourcePaths`.

We don't need to consciously abbreviate to avoid camel case. As for camel vs snake case, if we have to use multi word keys stick with camel.

Reply:
It's just saying `sourcePaths` is an array of `sourcePath`, same as `secrets` is an array of `secret`.  
From the `screwdriver.yaml` point of view, the key is always `sourcePaths` (with an `s`) (line 90)